contrib/facter.ps1

```diff
@@ -127,6 +127,7 @@ if ($buildSource) {
     "--with-regex",
     "--with-log",
     "--with-locale",
+    "--with-chrono",
```

Comment:
Yes. Some work for Solaris has already been done: https://github.com/puppetlabs/cfacter-build/blob/master/projects/boost/Makefile.SunOS-i386#L33